docs/manage/_advanced.rst

```diff
@@ -146,6 +146,83 @@ In case you need to create another **super admin** user, either to address addit
 You will be prompted to input the **Username** and **Password** and the new **super admin** user account will be created.
 
 
+Import Users from a CSV File
+****************************
+
+.. note:: 
+  This is currently an experimental feature, so please forward to the development team any details about the issues you may encounter while using it.
+
+  Command works on Kolibri version 0.9 and above.
+
+CSV File Structure
+""""""""""""""""""
+
+To import users into Kolibri with this command, you will need to provide the user data in a CSV (comma separated values) file format. You can export a CSV file from a tabular data in any spreadsheet program (Excel, Google Sheets, LibreOffice Calc, etc.).
+
+  .. figure:: img/csv.png
+      :alt: User data in a spreadsheet table
+
+      User data in a spreadsheet table.
+
+* Header row is optional, but if you do not include it, Kolibri will assume that you are providing the data in the following order:
+
+    ``<full_name>,<username>,<password>,<facility>,<class>``
+
+* If you do include a header row, you can provide less data, or put them a different order:
+
+    ``<full_name>,<username>,<password>``
+
+    ``<username>,<full_name>``
+
+* Only the ``username`` is required.
+
+* When you do not provide passwords for the imported users, Kolibri will set the default password ``kolibri`` for those usernames.
+
+* The facility can be either the facility name or the facility ID. If you do not provide the facility, Kolibri will import users in the default facility on the device. You can also specify the facility by adding the ``--facility`` argument in the command line (see below).
+
+
+Linux or OSX
+""""""""""""
+
+Open the Terminal and run this command from the folder where the CSV file is located:
+
+.. code-block:: bash
+
+  kolibri manage importusers your-csv-file.csv
+
+  kolibri manage importusers your-csv-file.csv --facility <your-facility>
+
+Windows
+"""""""
+
+* Kolibri version 0.9
+
+  .. code-block:: bash
+
+    C:\Python27\Scripts>kolibri manage importusers your-csv-file.csv
+    C:\Python27\Scripts>kolibri manage importusers your-csv-file.csv --facility <your-facility>
+
+* Kolibri version 0.10 or above
+
+  .. code-block:: bash
+
+    C:\Python34\Scripts>kolibri manage importusers name-of-your-csv-file.csv
+    C:\Python34\Scripts>kolibri manage importusers your-csv-file.csv --facility <your-facility>
```

Comment:
IIRC we have a general instruction somewhere about running commands in Windows, I think we should refer to this and not split up each and every CLI for Windows and *Nix?